x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ingest_pipelines/customize_pipeline_item.tsx

```diff
@@ -92,7 +81,7 @@ export const CustomizeIngestPipelineItem: React.FC = () => {
             </p>
           </EuiConfirmModal>
         )}
-        {hasIndexIngestionPipeline ? (
+        {hasIndexIngestionPipeline && (
           <EuiButtonEmpty
```

Comment:
Now that you're here anyway, maybe we should make this delete button red or give it a delete icon? What do you think @julianrosado?

Reply:
Julian would like this:
![image](https://user-images.githubusercontent.com/1972968/228600236-9ea45173-3646-41c5-a13a-f48aa20cd5b7.png)
I can work on that now.

Reply:
see updated screenshot with change :)